### PR TITLE
Fixed incorrect result for obj.Value().MSI() getter

### PR DESCRIPTION
**Pull request by earncef**
When an object is created as obj := objx.Map{...}
calling obj.Value().MSI() returns an empty map. It should return the object itself as map[string]interface{} instead

### CODE REVIEW

File: type_specific_codegen.go [hunk 1]
```diff
@@ -135,6 +135,9 @@ func (v *Value) MSI(optionalDefault ...map[string]interface{}) map[string]interf
 	if s, ok := v.data.(map[string]interface{}); ok {
 		return s
 	}
+	if s, ok := v.data.(Map); ok {
```
Comment (Hanzei) [inline type_specific_codegen.go]: Cloud you add a todo comment to find a better solution?

File: type_specific_codegen_test.go [hunk 2]
```diff
@@ -12,2339 +12,2339 @@ import (
   Tests for Inter (interface{} and []interface{})
 */
 func TestInter(t *testing.T) {
-  val := interface{}("something")
-  m := objx.Map{"value": val, "nothing": nil}
+	val := interface{}("something")
```
Comment (Hanzei) [inline type_specific_codegen_test.go]: Could you fix the indention in this file please?

Comment (earncef) [reply type_specific_codegen_test.go]: Isn't the indentation in this file more correct not vs what it was before, keeping consistent with the rest of the codebase? Even within the file itself the original indentation is inconsistent, using a full tab in some places and a half tab in others.

File: type_specific_codegen.go (hunk 1 above)
Comment (earncef) [reply type_specific_codegen.go]: Done

**Comment by earncef** [comment]
I will also try and work on a better solution for the type_specific_codegen.go file

File: type_specific_codegen_test.go (hunk 2 above)
Comment (Hanzei) [reply type_specific_codegen_test.go]: Your correct, but I would rather fix this in a other PR. See #69

**Comment by Hanzei** [comment]
I opened #70. You might want to look into this

File: type_specific_codegen_test.go (hunk 2 above)
Comment (earncef) [reply type_specific_codegen_test.go]: I've restored the indentation

**Pull request merged**